mobile/src/stateMachine/globalTriggers.ts

```diff
@@ -12,6 +12,7 @@ export enum GlobalTriggers {
     Settings,
     Profile,
     NotificationReceived,
+    HealthAuthSettings,
```

Comment:
Again, it would be nice if we keep the naming consistent here so if we are saying "Health Permissions Settings" elsewhere, ideally it would be called the same thing here. (Lmk if that doesn't make sense in this case)

Reply:
this is not used. now removed